src/north/CsvToHttp/CsvToHttp.class.js

```diff
@@ -174,12 +267,50 @@ class CsvToHttp extends ApiHandler {
           object.error.push(`Header "${mapping.httpField}": ${response.error}`)
         }
         object.value[field] = response.value
+      } else if (mapping.csvField.match(REGEX_CONTAIN_VARIABLE_STRING)) {
+        // split the input into a array of string
+        // "test ${value}" => [ "test ", ${value}]
+        const csvFieldSplit = mapping.csvField.split(REGEX_SPLIT_TEMPLATE_STRING).filter(Boolean)
+        const field = mapping.httpField
+        csvFieldSplit.forEach((element) => {
+          // match if the string starts with: ${...}
+          if (element.match(REGEX_MATCH_VARIABLE_STRING)) {
+            const headerToGet = element.match(REGEX_GET_VARIABLE)
+            headerToGet.forEach((header) => {
+              const response = CsvToHttp.convertToCorrectType(csvRowInJson[header], mapping.type)
+              if (response.error) {
+                object.error.push(`Header "${mapping.httpField}": ${response.error}`)
+              }
+              if (response.value) {
+                object.value[field] = CsvToHttp.insertValueInObject(object.value[field], response.value)
```

Comment:
what if we add an integer to a previous value which was a date (or other cases like string + integer, etc) ? Should you not test beforehand that the type are compatible and trigger a warning/error log otherwise ?

Reply:
My mistake it handles only string value (I'll change the conversion type)